chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -127,6 +127,17 @@
   options ['nodelaycompress','compress']
 end
 
+logrotate_app 'delayed_job-wca' do
+  path "#{repo_root}/WcaOnRails/log/delayed_job.log"
+  size "512M"
+  maxage 90
+  options ['nodelaycompress','compress']
```

Comment:
So now the log file options are duplicated. Would it be worth it to "dry them up"?

This might well be overkill. Just asking. Up to you :)